src/Activity.js

```diff
@@ -75,11 +60,21 @@ class Activity extends UserFitness {
     return [stepObj, this.user.friends[totalStepsPerFriend.indexOf(Math.max(...totalStepsPerFriend))]]
   }
 
-  returnThreeDayStepStreak() {
+  returnAverageForWeek(week, activityData) {
```

Comment:
Is this function copied in here twice? the function on line 26 looks identical - this might be a duplicate and need to be deleted?